src/dreyfus_index.erl

```diff
@@ -118,6 +119,17 @@ init({DbName, Index}) ->
             proc_lib:init_ack(Error)
     end.
 
+maybe_create_local_purge_doc(Db, IndexPid, Index) ->
+    DocId = dreyfus_util:get_local_purge_doc_id(Index#index.sig),
+    {ok, PurgeSeq} = clouseau_rpc:get_purge_seq(IndexPid),
```

Comment:
@chguocloudant We need to think here, if the initial purge_seq should be the database's purge_seq (or from clouseau).

Consider this, you are having maybe_create_local_purge_doc function for the upgrade. Before this upgrade clouseau was not storing any purge_seq (that what makes this index different from hastings). So, initially you would like to get purge_seq from a database.

Or if you have different plan for upgrade, please suggest it.

You can put line #124 after line #126, so to request purge_seq only when necessary.